plugins/qeta/src/components/QuestionsContainer/QuestionsContainer.tsx

```diff
@@ -87,6 +89,7 @@ export const QuestionsContainer = (props: QuestionsContainerProps) => {
 
   const onSearchQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     onPageChange(1);
+    analytics.captureEvent('search', event.target.value ?? '');
```

Comment:
The backstage search uses analytics with the same scope here: https://github.com/backstage/backstage/blob/master/plugins%2Fsearch-react%2Fsrc%2Fcontext%2FSearchContext.tsx#L141-L142

We don't want to mix Q&A analytics with that so I suggest changing this to `qeta_search` or something similar
 Also I suggest that you check the value is not empty or undefined before capturing event (like backstage search does)